doit/runner.py

```diff
@@ -395,6 +396,25 @@ def _run_start_processes(self, job_q, result_q):
         @param result_q: (multiprocessing.Queue) collect task results
         @return list of Process
         """
+        # #### DEBUG PICKLE ERRORS
```

Comment:
This debug code we can commented out. Looks like we might have more problems in the future :/